builtin/logical/aws/client.go

```diff
@@ -124,23 +187,22 @@ func (b *backend) getRootConfigs(ctx context.Context, s logical.Storage, clientT
 		opts = append(opts, awsutil.WithEnvironmentCredentials(false), awsutil.WithSharedCredentials(false))
 	}
 
-	// at this point, in the IAM case, regions contains nothing, and endpoints contains iam_endpoint, if it was set.
+	// at this point, in the IAM case, regions contains config.Region, if it was set.
+	//                                 endpoints contains iam_endpoint, if it was set.
```

Comment:
Where these comments meant to be combined into 1 statement?
Same situation in the comment right below this which also contains a few typos.

Reply:
i was trying to use tabulation to line up the different components, but i agree it makes it look odd.

Reply:
i also fixed a couple missing letters i apparently decided weren't cool enough to be included in the first draft of the comment.